sdk/identity/Azure.Identity/src/Azure.Identity.csproj

```diff
@@ -5,6 +5,7 @@
     <Version>1.6.0-beta.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>1.5.0</ApiCompatVersion>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net461</TargetFrameworks>
```

Comment:
I think you want to add this below otherwise you are going to just override your property changes here below.